src/nix/flake-archive.md

```diff
@@ -22,8 +22,20 @@ R""(
   # nix flake archive --json --dry-run nixops
   ```
 
+* Upload all flake inputs to a different machine for remote evaluation
+
+  ```
+  # nix flake archive --to ssh://some-machine
+  ```
+
+  On the remote machine the flake can be than access via it's store path. That computed like this:
```

Comment:
Few small typos here, I'd correct this to: 
"On the remote machine the flake can then be accessed via its store path. That's computed like this:"